doc/feature/pointDataEvolution.md

```diff
@@ -0,0 +1,85 @@
+Evolution of data at a point over time
+======================================
+
+One of the features that would make the tool extremely useful for scientists, is
+the ability to obtain the evolution of the data at a specific point over time.
+For example, they could load a current report and compute the LFP for a specific
+target, and once they have their full volume, they could focus on a specific
+point of the scene (e.g. a voxel), and obtain a 2D curve showing the evolution
+of voltage at that point over time.
```

Comment:
... for the whole time span. 
I think it should be clear that this feature has nothing to do with streaming, that is, is a post-mortem data processing.